lib/KCard/index.vue

```diff
@@ -109,38 +146,54 @@
 
   export default {
     name: 'KCard',
-    components: { BaseCard },
     props: {
       /**
-       * Sets card title if provided. The title should be
-       * unique and descriptive to aid searching through list of links.
+       * A regular Vue route object that defines
+       * where the card click should navigate.
        */
-      title: {
-        type: String,
-        required: false,
-        default: null,
+      to: {
+        type: Object,
+        required: true,
       },
       /**
-       * Sets the HTML heading level in range (h2 - h6) for the title .
+       * Sets the HTML heading level in range (h2 - h6) for the title.
+       * Ensure that the heading level is correct within a particular
+       * context of a page where cards are displayed.
        */
       headingLevel: {
         type: Number,
         required: true,
+        validator(value) {
+          if (value <= 6 && value >= 2) {
+            return true;
+          } else {
+            console.error(`[KCard] 'headingLevel' must be between 2 and 6.`);
+            return false;
+          }
+        },
       },
       /**
-       * An object containing the route definition for the link.
-       * Required and cannot be empty.
+       * Sets card title if provided. The title should be
+       * unique within a grid and descriptive to aid screenreader
+       * navigation.
        */
-      to: {
-        type: Object,
-        required: true,
+      title: {
+        type: String,
+        required: false,
+        default: null,
+      },
+      /**
+       * Specifies the number of lines allowed
+       * for the titlebefore truncation occurs.
```

Comment:
Just a little typo here
```suggestion
       * for the title before truncation occurs.
```

And I think is better to specify that is the _max_ number of lines before truncate it. I think right now it can be confused with "number of lines we will be reserving for every title" (so, if the title just have 1 line, there will be an empty line).

Reply:
I changed it to `Truncates title to this number of lines.`